x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/slm/SnapshotInvocationRecord.java

```diff
@@ -29,19 +31,23 @@
     implements Writeable, ToXContentObject, Diffable<SnapshotInvocationRecord> {
 
     static final ParseField SNAPSHOT_NAME = new ParseField("snapshot_name");
+    static final ParseField START_TIMESTAMP = new ParseField("snapshot_start_time")
+        .forRestApiVersion(RestApiVersion.onOrAfter(RestApiVersion.V_8));
     static final ParseField TIMESTAMP = new ParseField("time");
     static final ParseField DETAILS = new ParseField("details");
 
     private String snapshotName;
+    private long snapshotStartTimestamp;
     private long timestamp;
```

Comment:
Before I initially committed, I had renamed this field. But I was worried it would cause compatibility problems. Are you saying to rename it in  the state store as well? Or just change the name in this class, but keep it as "timestamp" in the state store?

Reply:
The ParseField can still be "time", but the internal variable name can be changed so it's clearer to use internally (and we can eventually change the XContent name, just in a separate change)